hammer_cli_katello.gemspec

```diff
@@ -74,7 +75,8 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'simplecov'
   gem.add_development_dependency 'mocha'
   gem.add_development_dependency 'ci_reporter'
-
-  gem.add_development_dependency "rubocop", "0.42"
-  gem.add_development_dependency "rubocop-checkstyle_formatter"
+  gem.add_development_dependency 'rubocop', '0.42'
```

Comment:
This is a seriously old version. It wouldn't surprise me if this wouldn't work on Ruby 3.